samples/accounts_user_links_batch_update_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+import accounts_user_links_batch_update
+
+
+FAKE_ACCOUNT_ID = "1"
+FAKE_ACCOUNT_USER_LINK_ID = "1"
+
+
+def test_accounts_user_links_batch_update():
```

Comment:
Same as others - would love to see a fake account that gets created and deleted in a fixture and updated in the test